client/jetpack-dashboard/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { makeJetpackDashboardLayout, render as clientRender } from 'controller';
+import {
+	antiSpam,
+	backups,
+	jetpackDashboard,
+	preloadJetpackDashboard,
+	scan,
+	security,
+	setupSidebar,
+} from './controller';
+
+export default function() {
+	page(
+		'/security/backups',
+		preloadJetpackDashboard,
+		setupSidebar,
+		backups,
+		makeJetpackDashboardLayout,
+		clientRender
```

Comment:
I would prefer to define `makeJetpackDashboardLayout` directly in this file. No need to inflate the shared `controller` module with it. Also, you can completely avoid the shared `controller` module and define your own thing:
```js
function render( context ) {
  ReactDom.render(
    <ReduxProvider store={ context.store }>
      <MomentProvider>
        <JetpackDashboardLayout primary={ context.primary } secondary={ context.secondary } />
      </MomentProvider>
    </ReduxProvider>,
    document.getElementById( 'wpcom' )
  );
);
```
...and use it in the page.js handler:
```js
page(
  '/security/backups',
  preloadJetpackDashboard,
  setupSidebar,
  backups,
  render
);
```